zstacklib/zstacklib/utils/gpu.py

```
@@ -1,6 +1,7 @@
 import os
 
 from zstacklib.utils import log, linux
+from zstacklib.utils.bash import *
```

Ruff (0.8.2)

`from zstacklib.utils.bash import *` used; unable to detect undefined names

(F403)